### DISCUSSION TITLE
Issue with vehicle’s ace options

### DISCUSSION THREAD

**Discussion by SOF-Ice** [Vdauphin/HeartsAndMinds#1471] [Question and answer] [answered]
Hi Vdauphin,
I would like to change the nemmera.

Its ok for the deleteloop etc ...

The Nemmera can refuel/rearm/repair.

I have already valid them and select a value (for rearm and refuel) with the "ace options" in eden for the new vehicle.
But refuel and rearm doesn't work (ace interactions don't appear) after the destruction and respawn of the vehicle.

Can you help me ?

**Comment by Vdauphin** [comment, marked as answer]
Hello,
See https://github.com/Vdauphin/HeartsAndMinds/pull/1472
Cheers
